ni_measurementlink_service/session_management.py

```diff
@@ -499,26 +515,28 @@ def unregister_sessions(self, session_info: Iterable[SessionInformation]) -> Non
     def reserve_all_registered_sessions(
         self, instrument_type_id: Optional[str] = None, timeout: Optional[float] = None
     ) -> MultiSessionReservation:
-        """Reserves and gets all sessions currently registered in the Session Manager.
+        """Reserve all sessions currently registered with the session management service.
 
         Args:
-            instrument_type_id (str): Instrument type ID for the measurement. If unspecified,
-                reserve sessions for all instrument types connected in the registered pin map
-                resource.
+            instrument_type_id: Instrument type ID for the measurement.
+
+                If unspecified, reserve sessions for all instrument types connected in the
+                registered pin map resource.
 
                 For NI instruments, use instrument type id constants, such as
                 :py:const:`INSTRUMENT_TYPE_NI_DCPOWER` or :py:const:`INSTRUMENT_TYPE_NI_DMM`.
 
                 For custom instruments, use the instrument type id defined in the pin map file.
 
-            timeout (float): Timeout in seconds. Allowed values,
-                0 (non-blocking, fails immediately if resources cannot be reserved),
-                -1 or negative (infinite timeout), or
-                any positive numeric value (wait for that number of second).
+            timeout: Timeout in seconds.
```

Comment:
Milliseconds

Reply:
The .proto files uses `timeout_in_milliseconds`. Maybe we should change to that for clarity. But that would be a breaking change, I guess.

Reply:
Oh, it looks from the tests like this is seconds. So maybe just fix the sentences below to be consistent.

Reply:
Other Python APIs like [threading.Lock](https://docs.python.org/3/library/threading.html#lock-objects) and [grpc calls](https://grpc.github.io/grpc/python/grpc.html#grpc.UnaryUnaryMultiCallable) use `timeout` in seconds, so the Python API is fine as it is. I just didn't notice the difference when fixing the formatting & removing "or negative" (which is not really allowed).

In the .proto file, we arguably should have used `google.protobuf.Duration`, but it is what it is.

Reply:
Also, now I see that while the session management service doesn't accept a timeout of -2, the Python API converts -2 to -1. Sigh.